.github/workflows/container.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Install apt dependencies
+        run: |
+          sudo apt-get install libpcap-dev libnl-genl-3-dev libnl-3-dev
```

Comment:
are the `libnl-genl-3-dev libnl-3-dev` required? the main lemming package doesn't depend on this

Reply:
Thanks - I think this is just for lemming, so there should be no requirements for them. Trimming. :-)